testing/visual-regression/backstop.json

```diff
@@ -714,6 +714,20 @@
       "expect": 0,
       "misMatchThreshold": 0.1,
       "requireSameDimensions": true
+    },
+    {
+      "label": "Components/OISC Warning/With title and description",
```

Comment:
One benefit of the new component docs structure is that there's no need to hide elements or use `a11yComponentToTest` as the Canvas is _only_ the component.